builders/build/templates/substrate.md

```diff
@@ -1,36 +1,36 @@
+description: The Tanssi repository includes a basic template that provides the necessary configuration to support the protocol and kick-start the development of an appchain.
 ---
 
 # Baseline Appchain Template {: #baseline-appchain-template }
 
 ## Introduction {: #introduction }
 
-The Tanssi repository includes a bare minimum Substrate template that provides the necessary configuration to support the Tanssi protocol and some essential modules, such as the one that allows handling the Tanssi appchain's currency.
+The Tanssi repository includes a bare minimum template that provides the necessary configuration to support the Tanssi protocol and some essential modules, such as the one that allows handling the Tanssi appchain's currency.
 
 This section covers this basic template, what it includes, and some aspects to consider when adding external dependencies.
 
 ## Baseline Appchain Template {: #baseline-appchain-template }
 
-Developing a Substrate-based runtime typically involves two primary steps:
+Developing an appchain runtime typically involves two primary steps:
 
-1. [Incorporating pre-existing Substrate built-in modules](/builders/build/customize/adding-built-in-module/){target=\_blank} into the runtime
+1. [Incorporating pre-existing built-in modules](/builders/build/customize/adding-built-in-module/){target=\_blank} into the runtime
 2. [Creating custom modules](/builders/build/customize/adding-custom-made-module/){target=\_blank} tailored to your specific application needs
 
-Since the provided template already includes the essential configurations for seamless integration into the Polkadot ecosystem and compatibility with the Tanssi protocol, teams interested in constructing an innovative Tanssi appchain can use this template as a starting point for adding their custom logic.
+Since the provided template already includes the essential configurations for seamless integration with the Tanssi protocol and the security provider (for example, [Symbiotic](https://symbiotic.fi/){target=\_blank} on Ethereum), teams interested in constructing an innovative Tanssi appchain can use this template as a starting point for adding their custom logic.
 
 Here are some of the features that come with this template:
 
 - Utilize Tanssi's [block production as a service](/learn/tanssi/technical-features/#block-production-as-a-service){target=\_blank}
-- Use [Polkadot's finality gadget](https://wiki.polkadot.network/docs/learn-consensus#finality-gadget-grandpa){target=\_blank}
-- Benefit from [Polkadot's shared security model](https://wiki.polkadot.network/docs/learn-parachains#shared-security){target=\_blank}
-- Use the [Polkadot.js API](/builders/toolkit/substrate-api/libraries/polkadot-js-api/){target=\_blank} to interact with the Substrate API
+- Leverage Ethereum-grade security from [Symbiotic](https://symbiotic.fi/){target=\_blank}
```

Comment:
Should we say here leverage Ethereum grade security? Or say more like "Choose the security provider of your choice, for example...."